server/routerlicious/packages/services-shared/src/storage.ts

```diff
@@ -282,7 +282,11 @@ export class DocumentStorage implements IDocumentStorage {
             // TODO: Make the rest endpoint handle this case.
             const opsContent = await gitManager.getContent(existingRef.object.sha, ".logTail/logTail");
             const ops = JSON.parse(
-                Buffer.from(opsContent.content, opsContent.encoding).toString()) as ISequencedDocumentMessage[];
+                            Buffer.from(
+                                opsContent.content,
+                                Buffer.isEncoding(opsContent.encoding) ? opsContent.encoding : null
```

Comment:
BufferEncoding is a string literal type. `type BufferEncoding = "ascii" | "utf8" | "utf-8" | "utf16le" | "ucs2" | "ucs-2" | "base64" | "latin1" | "binary" | "hex";` In @types/node v10 `encoding` param in `Buffer.from()` was a string, but in v12 it was changed to BufferEncoding type. Here it checks if the passed string is BufferEncoding or not, if not, then  pass null instead which will go to default value 'utf8'.

Reply:
Changed from null to undefined as it is optional parameter, `encoding` type should be BufferEncoding | undefined. If it is undefined, it will be default value 'utf8'.